mlflow/tracking/client.py

```diff
@@ -34,7 +34,20 @@ def __init__(self, tracking_uri=None):
         self.store = utils._get_store(self.tracking_uri)
 
     def get_run(self, run_id):
-        """:return: :py:class:`mlflow.entities.Run` associated with the run ID."""
+        """
+        Fetch the run from backend store. The resulting :py:class:`Run <mlflow.entities.Run>`
+        contains a collection of run metadata - :py:class:`RunInfo <mlflow.entities.RunInfo>`,
+        as well as a collection of run parameters, tags and metrics -
```

Comment:
Nit: parameters, tags and metrics -> parameters, tags, and metrics

Reply:
Added comma! Thanks!